mypy/test/data/check-modules.test

```diff
@@ -739,3 +739,18 @@ y = 42
 [builtins fixtures/module.py]
 [out]
 tmp/main.py:4: error: Unsupported left operand type for + ("int")
+
+[case testSuperclassInImportCycle]
```

Comment:
Yes, good point. This test is sensitive to the import order. I added a second test with the imports in the main module reversed.